src/Symfony/Component/String/Inflector/FrenchInflector.php

```diff
@@ -0,0 +1,257 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String\Inflector;
+
+/**
+ * French inflector.
+ *
+ * Be careful, this class does only inflect words, not adjectives nor composed words like "soixante-dix".
```

Comment:
~Be careful,~ This class does only inflect nouns; not adjectives nor composed words like "soixante-dix".